src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java

```diff
@@ -397,7 +397,8 @@ public void shouldNotAllowPatchDocumentByDocumentIdWhenServiceIsNotAuthorised()
     @Test
     public void shouldNotPatchMetaDataOnDocuments() {
         doThrow(ForbiddenException.class).when(documentManagementService).checkServicePermission(
-            eq(setDocumentMetaData()),
+            eq("BEFTA_CASETYPE_2_1"),
+            eq("BEFTA_JURISDICTION_2"),
```

Comment:
Use existing `BEFTA_JURISDICTION_2`

Reply:
This goes for all other instances in this class...

Reply:
cleaned up here 522b82c73c043bdbb8dedaa797594f26c15599e8